tests/e2e/normalize.go

```diff
@@ -106,6 +106,7 @@ func normalizeKRMObject(u *unstructured.Unstructured, project testgcp.GCPProject
 	// Specific to Compute
 	visitor.replacePaths[".status.observedState.certificateID"] = "1.719337333063698e+18"
 	visitor.replacePaths[".status.instanceId"] = "1111111111111111"
+	visitor.replacePaths[".status.gatewayId"] = "1.719337333063698e+18"
```

Comment:
Ouch ... this might be a problem if the GCP IDs overflow int64!

Reply:
Fixed in https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/2262

Should not break anything since we replace the value with fixed string here just for golden comparison, to avoid confusion I had a fix and ID should be an int64